code/modules/antagonists/malf_ai/malf_ai.dm

```diff
@@ -31,12 +33,13 @@
 	owner.special_role = job_rank
 	if(give_objectives)
 		forge_ai_objectives()
-
-	employer = pick(GLOB.ai_employers)
+	if(!employer)
+		employer = pick(GLOB.ai_employers)
 
 	malfunction_flavor = strings(MALFUNCTION_FLAVOR_FILE, employer)
 
-	add_law_zero()
+	if(give_zeroth_laws)
+		add_law_zero()
```

Comment:
This may interfere with late-rolling Malf after One-Human boards are used.

Reply:
What do you mean? Malf AIs that aren't created via this objective have the zeroth law variable set true on default, so it should behave the same as before my changes.